dags/pubmed.py

```diff
@@ -22,15 +22,22 @@
     command='make start pubmed 1900-01-01 2100-01-01'
 )
 
-processor_task = helpers.create_processor_task(
-    name='pubmed',
+unregistred_trials_task = helpers.create_processor_task(
+    name='pubmed_unregistered_trials',
     dag=dag
 )
 
-pubmed_linker_task = helpers.create_processor_task(
-    name='pubmed_linker',
+trials_remover_task = helpers.create_processor_task(
+    name='trial_remover',
     dag=dag
 )
 
-processor_task.set_upstream(collector_task)
-pubmed_linker_task.set_upstream(processor_task)
+publications_task = helpers.create_processor_task(
+    name='pubmed_publications',
+    dag=dag
+)
+
+
+unregistred_trials_task.set_upstream(collector_task)
+trials_remover_task.set_upstream(unregistred_trials_task)
+publications_task.set_upstream(trials_remover_task)
```

Comment:
@vitorbaptista 

> Why the `publication_task` depends on the `unregistered_trials` or the `trials_remover`?

It doesn't.
Both `publication_task`  and `unregistered_trials_task` depend only on `collector`.  I just placed them in this order because it seems the most efficient: `unregistered_trials_task` may remove some `records` of ex-unregistered trials  -> `trials_remover` processor deletes those ex-unregistered `trials` whose `records` were removed  -> `publications_tasks` merges `publications` with `trials` without wasting time on ex-unregistered trials that were already removed.
I though `set_upstream` implies only order not necessarily dependency. Do you know any way to make the distinction more obvious in the code?